apps/evm/ui/pool/ConcentratedLiquidityURLStateProvider.tsx

```diff
@@ -147,7 +147,7 @@ export const ConcentratedLiquidityURLStateProvider: FC<ConcentratedLiquidityURLS
             : state.token1.wrapped.address
           : undefined
 
-      const _searchParams = new URLSearchParams(searchParams)
+      const _searchParams = new URLSearchParams(Array.from(searchParams.entries()))
```

Comment:
Any reason for these changes?

Reply:
Was getting type errors because of ReadonlyUrlParams or something, found this as a solution on the nextjs github